prerendering.bs

```diff
@@ -950,6 +970,12 @@ To <dfn export>get the supported loading modes</dfn> for a [=response=] |respons
 
 Various behaviors are disallowed in [=prerendering navigables=] because they would be intrusive to the user, since the prerendered content is not being actively interacted with.
 
+<h3 id="pausing-script-execution">Pausing script execution</h3>
+
+If a [=navigable=]'s [=navigable/scripting mode=] is "`paused-until-activation`", the user agent must not execute script in the navigable's [=navigable/active document=]. This includes, but is not limited to executing `<script>` elements.
```

Comment:
This PR is more like the speculation-rules-side plumbing and doesn't define the behavior of "pausing script execution".

So, at least we should clearly label the to-be-defined concepts and centralize the undefined things (probably to the section).
Namely,  "Resume script execution" at Line 633.

Reply:
Thank you for the information!! So, I think we're going to discuss the blocking/unblocking implementation during TPAC. I will draft changes to scripting first and finalize it after TPAC.